js/base/Exchange.js

```diff
@@ -2146,11 +2146,11 @@ module.exports = class Exchange {
     handleSubTypeAndParams (methodName, market = undefined, params = {}) {
         let subType = undefined;
         // if set in params, it takes precedence
-        const subTypeInParams = this.safeString2 (params, 'subType', 'subType');
+        const subTypeInParams = this.safeString2 (params, 'subType', 'defaultSubType');
```

Comment:
just inversed orders , as `default` prefixed should have less priority as explicit keyword itself (imo).